noisifier/noisifier.py

```diff
@@ -0,0 +1,125 @@
+import os
+import random
+import itertools
+import click
+import librosa
+import numpy as np
+
+SUPPORTED_EXTENSIONS = [
+    '.wav',
+    '.flac',
+]
+
+
+class NoiseBank:
+    def __init__(self, config):
```

Comment:
Explicit is better than implicit.
https://www.python.org/dev/peps/pep-0020/

I would suggest passing explicit arguments instead of config